execution_workflows/DaPars/Dockerfile

```diff
@@ -10,9 +10,9 @@ ENV DEBIAN_FRONTEND noninteractive
 RUN apt-get update && apt-get install -y curl unzip python python-dev python3 python3-pip r-base bedtools wget
 
 # Install DaPars
-RUN curl -LJO gitreleases.dev/gh/ZhengXia/dapars/latest/dapars_0.9.1.zip
-RUN unzip -q "dapars_0.9.1.zip"
-RUN chmod +x /dapars/src/*
+RUN wget https://github.com/ZhengXia/dapars/archive/master.zip
```

Comment:
I don't feel very comfortable downloading the `master` branch since it could change without notice, meaning that it isn't reproducible. I think it's a good idea to follow best practices although It's true that it seems unlikely that master will change in the near future. In fact, DaPars does not seem to be actively developed, I don't know if DaPars2 superseded it. The last commit to the repo was made in July 2021 and it only updated the README whereas the last real change to the code was in November 2015, which fixed the issue you mention. It's unfortunate that the author didn't create a new release after that.

Unless you can convince him to create a new release or tag I would suggest downloading the latest commit [546bdd4](https://github.com/ZhengXia/dapars/commit/546bdd4d809b097fd7a946d42b56e0efa0228472) using the url https://github.com/ZhengXia/dapars/archive/546bdd4d809b097fd7a946d42b56e0efa0228472.zip.